bot/features/music/music.py

```diff
@@ -328,25 +328,29 @@ async def shuffle(self, ctx):
 
     @music.command(aliases=['v'])
     @TGACog.check_permissions()
-    async def volume(self, ctx, args):
+    async def volume(self, ctx, args=None):
         '''
         Adjusts the volume to the specified level.
         Where <args> is an integer between 0 and 100
         '''
         if self.voice_client and self.voice_client.is_connected():
-            try:
-                my_volume = int(args)
-                if my_volume >= 0 and my_volume <= 100:
-                    # Divide by 1000 because even at volume 1, it was always far too loud
-                    self.voice_client.source.volume = self.curr_volume = my_volume / 1000
-                else:
-                    raise Exception(ValueError)
-            except ValueError:
-                await ctx.message.channel.send(
-                    "For volume please enter a value between 0 and 100.")
-            except Exception as e:
-                await ctx.message.channel.send(f"An unknown error occured: {e}"
-                                               )
+            if not args:
+                current_volume = int(self.voice_client.source.volume * 1000)
+                await ctx.message.channel.send(f"Volume currently @ {current_volume}")
+            else:
+                try:
+                    my_volume = int(args)
+                    if my_volume >= 0 and my_volume <= 100:
+                        # Divide by 1000 because even at volume 1, it was always far too loud
+                        self.voice_client.source.volume = self.curr_volume = my_volume / 1000
+                    else:
+                        raise Exception(ValueError)
+                except ValueError:
```

Comment:
This exception raises a ValueError but it is not catching it correctly. Could you take a look at this?

Simply enter a volume > 100 for it to occur. It actually catches it below in the generalized Exception.

Reply:
This is fixed in the #72